contrib/kafka/pkg/dispatcher/dispatcher_test.go

```diff
@@ -309,7 +308,7 @@ func TestFromKafkaMessage(t *testing.T) {
 	}
 	got := fromKafkaMessage(kafkaMessage)
 	if diff := cmp.Diff(want, got); diff != "" {
-		t.Errorf("unexpected message (-want, +got) = %v", diff)
```

Comment:
are you sure you want the .String() version of the diff?

Reply:
`diff` _is_ a string (https://github.com/google/go-cmp/blob/master/cmp/compare.go#L98)